apps/test/unit/templates/manageStudents/MoveStudentsTest.js

```diff
@@ -126,12 +138,50 @@ describe('MoveStudents', () => {
         transferData={transferData}
         updateStudentTransfer={updateStudentTransfer}
         transferStudents={transferStudents}
+        cancelStudentTransfer={cancelStudentTransfer}
       />
     );
 
     wrapper.find('Button').simulate('click');
     expect(transferStudents.callCount).to.equal(0);
-    wrapper.find('#submit').simulate('click');
+    wrapper.find('#uitest-submit').simulate('click');
     expect(transferStudents.callCount).to.equal(1);
   });
+
+  it('calls cancelStudentTransfer on close', () => {
+    const wrapper = mount(
+      <MoveStudents
+        {...DEFAULT_PROPS}
+        updateStudentTransfer={updateStudentTransfer}
+        transferStudents={transferStudents}
+        cancelStudentTransfer={cancelStudentTransfer}
+      />
+    );
+
+    wrapper.find('Button').simulate('click');
+    expect(cancelStudentTransfer.callCount).to.equal(0);
```

Comment:
Tip: We've already got the [`sinon-chai`](https://github.com/domenic/sinon-chai) assertions enabled so you can make assertions about spies and stubs like this:

```js
expect(cancelStudentTransfer).not.to.have.been.called;
expect(cancelStudentTransfer).to.have.been.calledOnce;
```